### ISSUE TITLE
TravisCI builds failing with `fatal: Not a valid object name master`

### ISSUE THREAD

**Issue by Terseus** [dbcli/mycli#527]
## Description

All my TravisCI builds of mycli are failing with the message `fatal: Not a valid object name master` at the end of the log.

None of the tests fails.

This is very annoying when running the tests for a work-in-progress branch that it's not associated to a pull request (I think it works in pull requests branches? I'm not even sure) so I would be thankful if this is fixed somehow.

## Additional information

Here is an example failing job: https://travis-ci.org/Terseus/mycli/jobs/281719109

The problems looks to be a combination of how Travis clone the repositories and how we make the lint test...

Travis use the following command to clone a repo: `git clone --depth=50 --branch=<branch> https://github.com/<username>/<project>.git <username>/<project>`.
This commands creates a [shallow clone](https://www.git-scm.com/docs/git-clone#git-clone---depthltdepthgt) that therefores doesn't have a "master" branch (unless explicitly cloning and testing the "master" branch, of course).

On the other hand, the `lint` command test the code of a given branch, which by default [happens to be "master"](https://github.com/dbcli/mycli/blob/25291958ccec7cbd930e684dbcd0b89bdad63f72/tasks.py#L66).

Since there is no master branch in the shallow clone, the error occurs and the build fail.

## Tested solutions

I've done some tests in my branch [fix_pep8radius_default_rev](https://github.com/Terseus/mycli/tree/fix_pep8radius_default_rev).

We're not passing any parameter to the `lint` command so I though that passing it "HEAD" as the branch would fix it. I'm so innocent at times...

I think I broke the `lint` command because now **it never fail**, not matter what s**t you push (as you can see in my last commit of the previous mentioned branch).

## Closing thoughs

At this point it was clear to me that I don't understand the full details of what is supposed `pep8radius` to do, how to configure it or what parameters should I pass it, therefore I opened this issue with the hope that one of the main developers of the project can make a better decision than me.

PD.: Sorry for the wall of text, and sorry if this issue doesn't really make sense (e.g. if this is the expected behavior).

**Comment by tsroten** [comment]
@Terseus Yeah, this is a pain. It's because it's testing on branch updates -- where there is no `master` branch. You can fix this by turning off branch updates in your Travis CI settings for your mycli project.

We don't have branch updates turned on in the dbcli travis project for mycli, which is why this error doesn't affect the main project branches.

You could add `git fetch origin master` to your travis config file in mycli, maybe `before_install` or something.

**Comment by amjith** [comment]
Travis is no longer used.
